HccAna/interface/HccElectronTree.h

```diff
@@ -41,7 +41,7 @@
  #include "TLorentzVector.h"
  #include "FWCore/Framework/interface/LuminosityBlock.h"
  #include "FWCore/Framework/interface/Frameworkfwd.h"
- #include "FWCore/Framework/interface/EDAnalyzer.h"
+ //#include "FWCore/Framework/interface/EDAnalyzer.h"
```

Comment:
Can we simply remove this line? The same comment is valid for HccFinalLepTree.h, HccGenAna.h, HccHelper.h, HccJetTree.h, HccMuonAna.h, HccPhotonTree.h, HccSipAna.h, HccAna.cc

Reply:
Yes, I'll remove these lines